sep2tools/cert_validate.py

```diff
@@ -38,20 +38,20 @@ def validate_pem_certificate(cert_path: Path) -> bool:
     cert = x509.load_pem_x509_certificate(cert_data)
 
     # Check the validity period
-    current_time = datetime.utcnow()
-    if not cert.not_valid_before <= current_time:
+    current_time_utc = datetime.now(timezone.utc)
```

Comment:
I noticed in `cert_create.py` you're using a different module `from dateutil import tz` and `tz.UTC`.

But I saw from the documentation the `datetime` module has a `timezone` export that seems to do the same thing, not sure if I'm missing something since I'm not experienced with Python.

https://github.com/aguinane/SEP2-Tools/blob/5aab6b48d5c6bc0064344c8e3f386f096d444f56/sep2tools/cert_create.py#L342